src/swagger.ts

```diff
@@ -19,20 +17,26 @@ export type TContactObject = {
 	url: Option<string>;
 	email: Option<string>;
 };
-export const ContactObject: t.Type<TContactObject, mixed> = t.type({
-	name: stringOption,
-	url: stringOption,
-	email: stringOption,
-});
+export const ContactObject = t.type(
+	{
+		name: stringOption,
+		url: stringOption,
+		email: stringOption,
+	},
+	'TContactObject',
```

Comment:
do we need `T` prefix?

Reply:
We can change it in a separate `refactoring` PR.

Reply:
@scink https://github.com/devexperts/swagger-codegen-ts/issues/43